### PR TITLE
more isEmpty goodness

### PR DESCRIPTION
**Pull request by mhines-usgs**
@kmschoep-usgs -- just made a few more small changes for sensorreading

### CODE REVIEW

File: R/sensorreading-data.R [hunk 1]
```diff
@@ -47,7 +47,7 @@ formatSensorData <- function(data, columnNames){
     listElements <- data[listRows,]
     
     if ("displayTime" %in% names(data)) {
-      if(!is.na(listElements$displayTime) || is.null(listElements$time)) {
+      if(!isEmpty(listElements$displayTime) || isEmpty(listElements$time)) {
```
Comment (lindsayplatt) [inline R/sensorreading-data.R:50]: not sure this is doing what we would expect...especially for listElements$time. There might be a case where we can continue if its NA, but not NULL? Hence, it was only checking for NULL, but isEmpty checks for both. Which we had tests that looked at this unique condition...

Comment (lindsayplatt) [inline R/sensorreading-data.R:50]: @mhines-usgs @thongsav-usgs @dpattermann-usgs what are your thoughts on my previous comment? Should we be concerned?

Comment (mhines-usgs) [inline R/sensorreading-data.R:50]: I think you're right @lindsaycarr, I overlooked that this was operating on different variables!

Comment (lindsayplatt) [inline R/sensorreading-data.R:50]: could you update in your next PR? I'll likely not be merging anything anytime soon

Comment (lindsayplatt) [inline R/sensorreading-data.R:50]: oh wait, I see it here https://github.com/USGS-R/repgen/pull/333 !!!

**Comment by mhines-usgs** [comment]
yep, i'm waiting on it right now https://cida-eros-aqcudev.er.usgs.gov:8443/jenkins/job/Repgen%20DEV%20Deployer/381/console

**Pull request merged**